gitax/Visual Studio Projects/C Sharp Projects/GTXLibGit2Sharp/GTXLibGit2Sharp.csproj

```diff
@@ -84,17 +85,33 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
+    <AXProxy Include="Class.Global.axproxy">
+      <Generator>MSBuild:Compile</Generator>
+    </AXProxy>
     <AXProxy Include="Table.SysVersionControlTmpItem.axproxy">
       <Generator>MSBuild:Compile</Generator>
     </AXProxy>
   </ItemGroup>
+  <ItemGroup />
+  <ItemGroup>
+    <Content Include="lib\git2-e0902fb.dll" />
+    <Content Include="lib\LibGit2Sharp.dll" />
+    <Content Include="LibGit2Sharp.xml">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Import Project="$(MSBuildExtensionsPath)\Microsoft\DynamicsTools\Microsoft.Dynamics.Framework.Design.InteractionProject.targets" />
   <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
        Other similar extension points exist, see Microsoft.Common.targets.
   <Target Name="BeforeBuild">
-  </Target>
+  </Target>-->
   <Target Name="AfterBuild">
-  </Target>
-  -->
+    <Message Text="Copying Git libraries to Client" Importance="normal" />
+	<CreateItem Include="$(ProjectDir)lib\*.*">
+		<Output TaskParameter="Include" ItemName="FilesToArchive"/>
+    </CreateItem>
+    <Copy SourceFiles="@(FilesToArchive)" DestinationFolder="$(AxClientBin)%(FilesToArchive.RecursiveDir)" ContinueOnError="true"/>	
+	</Target>
```

Comment:
One thing I have noticed here - is this VS 2013, meaning, coming from AX 2012 R3 CU8?